.github/workflows/test.yml

```diff
@@ -19,14 +19,12 @@ jobs:
         run: npm cache clean -f
       - name: install dependencies
         run: npm ci
+      - name: preload sha check 1
+        run: echo "PRELOAD_SHASUM_ORIGIN=$(shasum packages/app/src/preloadApi.ts)" >> $GITHUB_ENV
       - name: build
         run: npm run build
       - name: tests
         run: npm run test
-      - name: preload sha check 1
-        run: echo "PRELOAD_SHASUM_ORIGIN=$(shasum packages/app/src/preloadApi.ts)" >> $GITHUB_ENV
-      - name: run preload generation
-        run: npm run gen:app:preload
```

Comment:
this one got removed. intentional?

Reply:
yeah -- the sha is checked even earlier now and the preload build happens alongside `npm run build` automatically now